java-checks/src/main/java/org/sonar/java/checks/CycleBetweenPackagesCheck.java

```diff
@@ -23,6 +23,7 @@
 import org.sonar.api.server.rule.RulesDefinition;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.sonar.api.server.rule.RulesDefinition'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AUselessImportCheck)